cmd/webhook/app/options/options.go

```diff
@@ -67,6 +68,9 @@ func (o *WebhookOptions) AddFlags(fs *pflag.FlagSet) {
 	fs.StringVar(&o.DynamicServingCASecretName, "dynamic-serving-ca-secret-name", "", "name of the secret used to store the CA that signs serving certificates certificates")
 	fs.StringSliceVar(&o.DynamicServingDNSNames, "dynamic-serving-dns-names", []string{""}, "DNS names that should be present on certificates generated by the dynamic serving CA")
 	fs.StringVar(&o.Kubeconfig, "kubeconfig", "", "optional path to the kubeconfig used to connect to the apiserver. If not specified, in-cluster-config will be used")
+	fs.StringVar(&o.APIServerHost, "master", "", ""+
```

Comment:
Is `master` still the recommended terminology/name for this flag? I know that the naming working group has been looking to change uses of this term

Reply:
Yes, I agree. I'll have a look at what they are using/migrating to. At the least, this should be changed to `--api-server-host` or something similiar.

Reply:
Looks like it is still called the same upstream https://github.com/kubernetes/kubernetes/blob/b11d0fbdd58394a62622787b38e98a620df82750/cmd/kube-controller-manager/app/options/options.go#L266

Will rename to `--api-server-host`